### PR TITLE
{Core} Bump azure.mgmt.resource.subscriptions to 2019-11-01

### LINKED ISSUE #11949: GeographyGroup is missed in az account list-locations result
### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.80

**Describe the bug**
GeographyGroup is missed in az account list-locations result

**To Reproduce**
az account list-locations

**Expected behavior**
GeographyGroup info for each region should be return

**Environment summary**
cli version: 2.0.80

**Additional context**
This seems introduced by recent REST api upgrade, cli should be updated to include these new location metadatas

### PR DESCRIPTION
**Pull request by jiasli**
**Description**  
Fix #11949: [`geographyGroup`](https://docs.microsoft.com/en-us/rest/api/resources/subscriptions/listlocations#locationmetadata) is missing from `az account list-locations`

This PR bumps `azure.mgmt.resource.subscriptions` to `2019-11-01` which was released in https://github.com/Azure/azure-sdk-for-python/pull/11482.

**Testing Guide**
```
> az account list-locations
[
  {
    "displayName": "East US",
    "id": "/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/locations/eastus",
    "metadata": {
      "geographyGroup": "US",          <<<<<<<<<<<<<<<<<<
      "latitude": "37.3719",
      "longitude": "-79.8164",
      "pairedRegion": [
        {
          "id": "/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/locations/westus",
          "name": "westus",
          "subscriptionId": null
        }
      ],
      "physicalLocation": "Virginia",
      "regionCategory": "Recommended",
      "regionType": "Physical"
    },
    "name": "eastus",
    "regionalDisplayName": "(US) East US",
    "subscriptionId": null
  },
  ...
```

### CODE REVIEW

**Comment by yonzhan** [comment]
Bump subscription

**Comment by jiasli** [comment]
The CI result is weird:

> ❌ Azure.azure-cli (Automation Test (Profile Latest) Python36) 
> ✔ Azure.azure-cli (Automation Test (Profile Latest) Python38)
> 
> ❌ CLI Automation Full Test (Automation Test Python36) 
> ❌ CLI Automation Full Test (Automation Test Python38) 

The failed test is `VMImageListThruServiceScenarioTest.test_vm_images_list_thru_services`. The recording was updated by live-running the test again.

It passes on Windows Python 3.8.3 but fails on Linux Python 3.8.2.

```
        except CannotOverwriteExistingCassetteException as ex:
>           raise AssertionError(ex)
E           AssertionError: No match for the request (<Request (GET) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/westus/publishers/canonical-test/artifacttypes/vmimage/offers?api-version=2019-07-01>) was found. Can't overwrite existing cassette ('/home/jladmin/azure-cli/src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vm_images_list_thru_services.yaml') in your current record mode ('once').

azure-cli/src/azure-cli-testsdk/azure/cli/testsdk/base.py:294: AssertionError
```

@qwordy, could you help take a look?

**Comment by qwordy** [comment]
I am looking at it. I have reproduced this issue.

File: src/azure-cli/azure/cli/command_modules/vm/tests/hybrid_2018_03_01/test_vm_commands.py [hunk 1]
```diff
@@ -64,6 +64,8 @@ def test_vm_usage(self):
 
 class VMImageListThruServiceScenarioTest(ScenarioTest):
 
+    # TODO: Sometimes fail in playback mode. Temporarily make it live only. #14292
+    @live_only()
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/vm/tests/hybrid_2018_03_01/test_vm_commands.py:68]: Will address in https://github.com/Azure/azure-cli/issues/14292.

**Pull request merged**